apps/editing-toolkit/tsconfig.json

```diff
@@ -30,7 +30,7 @@
 
 		// Some type helpers from our webpack/
 		"typeRoots": [ "./typings", "./node_modules/@types", "../../node_modules/@types" ],
-		"types": [ "jest", "@emotion/core" ]
+		"types": [ "jest", "@emotion/react" ]
```

Comment:
Not for this PR, but I suspect we can remove `@emotion/react` from here. Probably even remove `types` entirely.